capa/main.py

```diff
@@ -706,9 +712,9 @@ def collect_metadata(
     if rules_path != [RULES_PATH_DEFAULT_STRING]:
         rules_path = [os.path.abspath(os.path.normpath(r)) for r in rules_path]
 
-    format_ = get_format(sample_path)
+    format_ = get_format(sample_path) if format_ == FORMAT_AUTO else f"{format_} (manual)"
     arch = get_arch(sample_path)
-    os_ = get_os(sample_path)
+    os_ = get_os(sample_path) if os_ == OS_AUTO else f"{os_} (manual)"
```

Comment:
i see what you're doing here but im not sure if this is the best way to do it.

i think its good to record that the user overrode the autodetection of format and os. in this part of the code, we're still dealing with structured data, not rendered strings. so, im inclined to store this as something like:

```py
{format: {name: "PE", override: False}}
```

and then render this only at the last moment possible. this way, an automated system can operate over the data without having to do `if "(manual)" in format and format.startswith("PE"): ...`.

that being said, either way this is a breaking change to our file format (though a slight one), so let's make extra sure we want to make this change.

thoughts @mr-tz @mike-hunhoff ?

Reply:
Distinguishing auto and manual could be neat, although I'm not sure it's really necessary.
What about the other places where `get_format` and `get_os` are called? Should we pass another argument to them and handle this there?

Reply:
the result document stores the argv used to invoke capa, so perhaps users can inspect those fields to determine if the OS/format/etc. were overridden

Reply:
> What about the other places where get_format and get_os are called?

looks like its only `main.py` today

Reply:
@captainGeech42 what do you think of inspecting `metadata.argv` to determine if the os/format was overridden? if its ok, please review changes here: https://github.com/mandiant/capa/pull/1350/commits/98eb28704cf6962dbe111c32b10d60f08a98b528